package.json

```diff
@@ -30,7 +30,7 @@
     "dev:backend:search-loader": "yarn workspace @hashintel/hash-search-loader dev",
     "dev:frontend": "yarn workspace @hashintel/hash-frontend dev",
     "exe": "ts-node --require dotenv-flow/config --transpile-only",
-    "external-services": "docker-compose --file packages/hash/external-services/docker-compose.yml --env-file .env",
+    "external-services": "docker compose --file packages/hash/external-services/docker-compose.yml --env-file .env",
```

Comment:
This took me about two hours because `docker-compose` is an alias to `docker compose` for macOS. On GH Actions, `docker-compose up --wait` will fail, but `docker compose up --wait` will not. 😒 😡

Reply:
God that is so frustrating